### DISCUSSION TITLE
Doris 能否支持存储过程？

### DISCUSSION THREAD

**Discussion by XinSuiNian** [apache/doris#38063] [A - General / Q&A]
目前使用的数据是 PG ，PG 中有使用了存储过程做了一些逻辑处理，现在想迁移到Doris上来，但是观看了Doris的文档，并没有发现有关存储过程相关的文档。所以想咨询一下：
1、Doris 能支持存储过程？如果支持，有没有一些文档可以看？
2、在Doris的文档中发现了变量的限制，不能通过变量给变量赋值。如果Doris 支持存储过程的话，能否在存储过程中 使用变量对变量赋值？

**Comment by XinSuiNian** [comment]
补充提问：另外 如果Doris 不支持存储过程的话，有什么好的方式，能实现 平滑的替代存储过程的功能？

**Comment by ixzc** [comment]
中文问题在ask.selectdb.com提个帖子
